test.js

```diff
@@ -318,6 +318,16 @@ test('result.killed is false if not killed, in sync mode', t => {
 	t.false(result.killed);
 });
 
+test('result.killed is false on process error', async t => {
+	const {killed} = await t.throwsAsync(execa('wrong command'));
+	t.false(killed);
+});
+
+test('result.killed is false on process error, in sync mode', t => {
+	const {killed} = t.throws(() => execa.sync('wrong command'));
```

Comment:
Don't use inline arrow function here.

Reply:
Should I use an inline non-arrow function instead? Should I use a `try`/`catch` block?

Reply:
No, it should be like:

```js
const {killed} = t.throws(() => {
	execa.sync('wrong command');
});
```

Reply:
Fixed. I also fixed it in two other places where this was done in the tests.